### ISSUE TITLE
Dash "-" in Ticker() name works fine but not when in Tickers()

### ISSUE THREAD

**Issue by aglove2189** [ranaroussi/yfinance#641]
``` python
import yfinance as yf

yf.Ticker("BRK-B").history()  # works

yf.Tickers(["SPY", "BRK-B"]).history()  # AttributeError
```

## Output
```
[*********************100%***********************]  2 of 2 completed
Traceback (most recent call last):

  File "<ipython-input-21-0337fe26029c>", line 1, in <module>
    yf.Tickers(["SPY", "BRK-B"]).history()

  File "C:\Users\user\AppData\Local\Continuum\miniconda3\lib\site-packages\yfinance\tickers.py", line 64, in history
    return self.download(

  File "C:\Users\user\AppData\Local\Continuum\miniconda3\lib\site-packages\yfinance\tickers.py", line 91, in download
    getattr(self.tickers, symbol)._history = data[symbol]

AttributeError: 'Tickers' object has no attribute 'BRK-B'
```

Looks like the namedtuple `self.tickers` in `Tickers()` renames 'dashed' tickers, here's a minimal example:

```python
from collections import namedtuple as _namedtuple

ticker_objects = {}
for ticker in ["SPY", "BRK-B"]:
    ticker_objects[ticker] = yf.Ticker(ticker)

tickers = _namedtuple(
            "Tickers", ticker_objects.keys(), rename=True
        )(*ticker_objects.values())
print(tickers)

>>> Tickers(SPY=yfinance.Ticker object <SPY>, _1=yfinance.Ticker object <BRK-B>)
```

The AttributeError happens on line 90 when `getattr` is called, not sure if this loop is needed but removing fixes the problem:
```python
for symbol in self.symbols:
    getattr(self.tickers, symbol)._history = data[symbol]
```

**Comment by gprossignoli** [comment]
I think this issue is strongly related to my PR: #613

**Comment by robertbetts** [comment]
A Named Tuple is used for the Tickers().tickers class property and the the inputed tickers are assigned as tuple named keys. these named keys will be subject to the the Python variable name policy.

> Officially, variable names in Python can be any length and can consist of uppercase and lowercase letters ( A-Z , a-z ), digits ( 0-9 ), and the underscore character ( _ ). An additional restriction is that, although a variable name can contain digits, the first character of a variable name cannot be a digit.

I run into this issue with the following tickers: 3IN, 888, ITV.L. These are all in the FTSE 250.

**Comment by ranaroussi** [comment]
Fixed in 0.1.57
